include/SDL_config.h

```diff
@@ -29,7 +29,9 @@
  */
 
 /* Add any platform that doesn't build using the configure system. */
-#if defined(__WIN32__)
+#if defined(PLAYDATE)
+#include "SDL_config_playdate.h"
+#elif defined(__WIN32__)
```

Comment:
added this to the top so it gets caught before host platforms - the config file unsets platform specific flags